src/index.js

```diff
@@ -63,6 +64,9 @@ class Vimeo extends React.Component {
         case 'color':
           player.setColor(value);
           break;
+        case 'controls':
+          player.controls(value);
```

Comment:
does this actually exist? I think you can't change the `controls` value on the fly with the vimeo SDK

Reply:
That's right @goto-bus-stop I've just removed, thanks for the catch 👍